src/out/build/x64-Debug/CMakeFiles/3.19.20122902-MSVC_2/CMakeCCompiler.cmake

```diff
@@ -0,0 +1,77 @@
+set(CMAKE_C_COMPILER "D:/ProgramFiles/VisualStudio/VC/Tools/MSVC/14.28.29910/bin/Hostx64/x64/cl.exe")

```

Comment:
All these build files in the `out` directory have been accidentally added to this PR and first need to be deleted from this PR . It is recommended that all build files are created in a `build` dir as this is already included in the `.gitignore` file. If need be we could add an `out` directory the git `.ignore` file if that is a common output dir used with MSVC compiler.